packages/template-retail-react-app/app/contexts/index.js

```diff
@@ -40,65 +40,51 @@ export const CategoriesProvider = ({treeRoot = {}, children, locale}) => {
 
     const api = useCommerceAPI()
     const [root, setRoot] = useState({
-        // map over the server-provided cat
+        // Clone the server-provided category
```

Comment:
I don't remember what `useState`'s internals are like. Is cloning `treeRoot` necessary or is it a vestige of when we mutated the data?

Reply:
We still need to copy tree root just in terms of React internals and object observability, you want to mutate the whole object